src/dft/backends/rocfft/commit.cpp

```diff
@@ -366,10 +376,10 @@ class rocfft_commit final : public dft::detail::commit_impl<prec, dom> {
                                                         nullptr, // in offsets
```

Comment:
Can `plan_desc` be safely re-used here if it was already used above in `valid_forward`? No memory leak to worry about for instance?

Reply:
I think so:
>The plan description object can be safely destroyed after it is given to the `rocfft_plan_create()`

https://rocm.docs.amd.com/projects/rocFFT/en/latest/reference/api.html#plan-description